docs/content/docs/test/Run.md

```diff
@@ -1,39 +1,111 @@
 ---
-title: Введение
+title: Запуск тестов
 type: docs
 ---
 
 # Запуск тестов
 
-## IDE
+{{<avito page>}}
+
+В данном документе собраны способы запуска unit и instrumentation тестов. Описаны наборы тестов, которые запускаются
+автоматически, а также способы позволяющие запустить нужные тесты в ручном режиме.
+
+## Автоматически запускаемые тесты
+
+Существует несколько наборов тестов, запускаемых автоматически.
+
+### Тесты на PR
+
+При создании Pull Request'а ваша ветка ребейзится на целевую ветку и
+[автоматически запускаются](http://links.k.avito.ru/PV):
+- Юнит тесты
+- Инструментальные тесты:
+  - Перформанс
+  - Компонентные
+  - [Скриншотные]({{< ref "/docs/test/ScreenshotTesting.md" >}})
+  - Интеграционные (`@IntegrationTest`)
+- [Android Lint]({{< ref "/docs/checks/AndroidLint.md" >}})
+
+За подробностями можно обратиться к конфигурации `fastCheck` в `avito/build.gradle`.
+
+### Проверки develop'а
+
+Раз в 3 часа при наличии изменений на develop-ветке [запускаются](http://links.k.avito.ru/5u) следующие проверки:
+- Юнит тесты
+- Инструментальные тесты:
+  - Компонентные
+  - E2E
+  - [Скриншотные]({{< ref "/docs/test/ScreenshotTesting.md" >}})
+  - Интеграционные (`@IntegrationTest`)
+  - Ручные (`@ManualTest`)
+- [Android Lint]({{< ref "/docs/checks/AndroidLint.md" >}})
+
+**ВАЖНО!** Тесты, почеменные как [`@Flaky`]({{< ref "/docs/test/FlakyTests.md" >}}) запускаются, но на результат прогона
+не влияют.
+
+За подробностями можно обратиться к конфигурации `fullCheck` в `avito/build.gradle`.
+
+### Регресс
+
+Регресс [автоматически запускается](http://links.k.avito.ru/OjE) каждый день со вторника по субботу в 5 и 8 утра, а
+также каждые 4 часа в понедельник и воскресенье. Результаты прогонов агрегируются в отчет по коммиту, с которого был
+запущен регресс, и упавшие тесты рассылаются ответственным юнитам в соответствующие чаты в Slack.
+
+Регресс включает в себя:
+- Юнит тесты
+- Инструментальные тесты:
+  - Компонентные
+  - E2E
+  - [Скриншотные]({{< ref "/docs/test/ScreenshotTesting.md" >}})
+  - Интеграционные (`@IntegrationTest`)
+  - Ручные (`@ManualTest`)
+- [Android Lint]({{< ref "/docs/checks/AndroidLint.md" >}})
+
+За подробностями можно обратиться к конфигурации `release` в `avito/build.gradle`.
+
+## Запуск тестов вручную
+
+### IDE
 
 Обрати внимание на [кастомные конфигурации]({{< ref "/docs/ide/Configurations.md" >}})
 
-### Robolectric
+#### Robolectric
 
 Просто запускаем стрелками Run напротив имени тестового класса или метода.
 
-Важно чтобы в настройках запуск был делегирован Gradle:
+#### Instrumentation
+
+Аналогично Robolectric тестам - просто запускаем тест средствами Android Studio.
+
+### Запуск всех E2E тестов
```

Comment:
Can we move it below the next one? I think our users should see a preferable way to run tests earlier

Reply:
Moved